2022q4/azure.adoc

```diff
@@ -0,0 +1,42 @@
+=== FreeBSD on Microsoft HyperV and Azure
+
+Links: +
+link:https://wiki.freebsd.org/MicrosoftAzure[Microsoft Azure article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/MicrosoftAzure[] +
+link:https://wiki.freebsd.org/HyperV[Microsoft HyperV article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/HyperV[]
```

Comment:
`URL: link:https://wiki.freebsd.org/MicrosoftAzure[https://wiki.freebsd.org/MicrosoftAzure]` and likewise for the next one.